test/integration_test.go

```diff
@@ -2,15 +2,20 @@ package test
 
 import (
 	"fmt"
+	"os"
 	"testing"
 
 	"github.com/sul-dlss-labs/permissions-service/config"
 	baloo "gopkg.in/h2non/baloo.v3"
 )
 
 func setupTest() *baloo.Client {
-	port := config.NewConfig().Port
-	return baloo.New(fmt.Sprintf("http://localhost:%v", port))
+	remoteHost, ok := os.LookupEnv("TEST_REMOTE_ENDPOINT")
```

Comment:
I thought we changed the variable to `TEST_REMOTE_ENDPOINT` so that it would contain both the `HOST` and `PORT`.

Reply:
That's true. That's what this does no?

Reply:
Ah right, I was thinking TEST_REMOTE_ENDPOINT had a default elsewhere, but, I guess it doesn't.